explorer/ExplorerControls.tsx

```diff
@@ -140,6 +140,7 @@ export class ExplorerControlPanel extends React.Component<{
                 }}
                 styles={styles}
                 isSearchable={options.length > 20}
+                maxMenuHeight={350}
```

Comment:
fyi, the default value here is 300. i chose 350 because it's a little higher, but not too much, and makes the scrollbar disappear for now.

Reply:
anything against making 350 the default?

Reply:
what do you mean?
do you mean that all react-selects should have that maxHeight, or what the downsides of this one change are?

Reply:
Ah I somehow assumed that default was in one of our components, and thought it might be a chance to re-evaluate our default value if it made more sense to have those extra 50 px in all places it is used. So effectively, I meant (a variation of) the first one.